services/admin/backend/service/project/project_test.go

```diff
@@ -63,11 +62,78 @@ func TestProject_CreateProject(t *testing.T) {
 	// get project
 	foundProject, err := service.GetProject(ctx, projectId)
 	assert.Nil(t, err)
+
 	assert.Equal(t, expectedAggregateType, foundProject.AggregateType().String())
 	assert.Equal(t, expectedShortCode, foundProject.ShortCode().String())
 	assert.Equal(t, expectedShortName, foundProject.ShortName().String())
 	assert.Equal(t, expectedLongName, foundProject.LongName().String())
 	assert.Equal(t, expectedDescription, foundProject.Description().String())
+}
+
+func TestService_CreateProject_ExistingShortCodeError(t *testing.T) {
```

Comment:
General question about test functions naming convention. I like the informative part, but maybe we don't need the 'TestService' part?

Reply:
Yeah I'm not sure about it but IntelliJ always suggests "Test[ParentDirectory]_[MethodName]". I added the part at the end about the error. The first part is actually quite helpful in finding the failing test in the code.